ocp_resources/virtual_machine_clone.py

```diff
@@ -0,0 +1,82 @@
+from ocp_resources.constants import TIMEOUT_4MINUTES
+from ocp_resources.resource import NamespacedResource
+from ocp_resources.utils import TimeoutSampler
+from ocp_resources.virtual_machine import VirtualMachine
+
+
+class VirtualMachineClone(NamespacedResource):
+    api_group = NamespacedResource.ApiGroup.CLONE_KUBEVIRT_IO
+
+    def __init__(
+        self,
+        name=None,
```

Comment:
You need to remove `name`, `namespace` and `yaml_file` from here (and from the docstring)

Reply:
done